tests/monotouch-test/Security/CertificateTest.cs

```diff
@@ -555,7 +564,12 @@ public void MailX1 ()
 		[Ignore ("https://bugzilla.xamarin.com/show_bug.cgi?id=39952")]
 		public void MailX2 ()
 		{
-			using (var cert = new X509Certificate2 (mail_google_com)) {
+#if MONOMAC // Not disposable on mac
```

Comment:
That looks like a (very strange) XM bug, because X509Certificate is disposable according to MSDN: https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.x509certificate(v=vs.110).aspx